### DISCUSSION TITLE
Welcome to GseaVis Discussions!

### DISCUSSION THREAD

**Discussion by junjunlab** [junjunlab/GseaVis#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xianjian4** [comment]
junjunlab教授：
您好！
我今天在网络上惊喜的发现您修改的“GseaVis 多条通路可视化”代码，在进行尝试的时候发现，这部分只能使用您的测试数据，载入不了自己的数据，特向您请问解决方法！
若您能解答，真是万分感谢！祝您工作顺利、生活愉快！

**Reply by junjunlab** [reply]
请提供具体的代码和报错信息

**Reply by xianjian4** [reply]
俊俊教授您好！
原代码是# load data
test_data <- system.file("extdata", "gseaRes.RDS", package = "GseaVis")
gseaRes <- readRDS(test_data)
我按您给的测试文件可以完美运行，无任何差错。
然而，我现有的文件信息是GSEA软件导出的tsv表格，我想请问extdata和gseaRes.RDS是都需要替换吗？我tsv转的RDS文件，无法替换运行

**Comment by 1363360897** [comment]
请问，怎么调节坐标轴标签或者刻度大小呢？
